great_expectations/rule_based_profiler/types/data_assistant_result/volume_data_assistant_result.py

```diff
@@ -41,29 +41,38 @@ def plot(
                 "You may either use `include_column_names` or `exclude_column_names` (but not both)."
             )
 
-        charts: List[alt.Chart] = []
+        display_charts: List[alt.Chart] = []
+        return_charts: List[alt.Chart] = []
 
         expectation_configurations: List[
             ExpectationConfiguration
         ] = self.expectation_suite.expectations
 
-        table_domain_charts: List[alt.Chart] = self._plot_table_domain_charts(
+        table_domain_charts: List[alt.Chart]
+        (table_domain_charts,) = self._plot_table_domain_charts(
             expectation_configurations=expectation_configurations,
             prescriptive=prescriptive,
         )
-        charts.extend(table_domain_charts)
-
-        column_domain_chart: List[alt.Chart] = self._plot_column_domain_charts(
+        display_charts.extend([table_domain_charts])
+        return_charts.extend([table_domain_charts])
+
+        column_domain_display_charts: List[alt.Chart]
+        column_domain_return_charts: List[alt.Chart]
+        (
+            column_domain_display_charts,
+            column_domain_return_charts,
+        ) = self._plot_column_domain_charts(
             expectation_configurations=expectation_configurations,
             include_column_names=include_column_names,
             exclude_column_names=exclude_column_names,
             prescriptive=prescriptive,
         )
-        charts.extend(column_domain_chart)
+        display_charts.extend([column_domain_display_charts])
+        return_charts.extend([column_domain_return_charts])
```

Comment:
We'd need a conditional to ensure that we use the base display for single column tables right?

Reply:
Also is there a reason we're putting these in lists? Aren't they already lists?